src/shared/ResourceForm/components/CollapsibleSection.js

```diff
@@ -54,7 +55,9 @@ export function CollapsibleSection({
       <header
         onClick={toggle}
         aria-label={`expand ${title}`}
-        style={{ marginLeft: `${nestingLevel * 16}px` }}
+        style={{
+          marginLeft: `calc(${nestingLevel} * ${spacing.sapUiSmallMarginBegin.marginLeft})`,
+        }}
```

Comment:
wow, is it really necessary?

Reply:
We probably will change it in the next task but setting proper margin for nested form sections was wild